lib/doorkeeper/orm/active_record.rb

```diff
@@ -12,6 +12,27 @@ def self.initialize_models!
           end
         end
       end
+
+      def self.initialize_application_owner!
+        require 'doorkeeper/models/concerns/ownership'
+
+        Doorkeeper::Application.send :include, Doorkeeper::Models::Ownership
+      end
+
+      def self.check_requirements(_)
```

Comment:
If this method doesn't take any arguments, we can remove the `(_)`. If it does, let's name it for what it is (`(_variable_name)`, for example).